Index: app/src/main/java/com/turingoal/mengbao/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/turingoal/mengbao/MainActivity.java	(date 1510722847000)
+++ app/src/main/java/com/turingoal/mengbao/MainActivity.java	(revision )
@@ -9,7 +9,7 @@
 
 public class MainActivity extends AppCompatActivity {
 
-    private TextView mTextMessage; //
+    private TextView  mTextMessage; //
 
     private BottomNavigationView.OnNavigationItemSelectedListener mOnNavigationItemSelectedListener
             = new BottomNavigationView.OnNavigationItemSelectedListener() {
